### DISCUSSION TITLE
A word of thanks for my first successful replication backup

### DISCUSSION THREAD

**Discussion by hput** [psy0rz/zfs_autobackup#138] [Show and tell]
I'm about to embark on a mission to try to repair a 15/22 sata socket on a HP Z800, so in gearing up to do it, I thought I should pull a replication backup on the problem host.  At least the rpool. 

I really think a word or two of thanks are in order for the folks on this forum for taking an aging greenhorn whose pea brain has been leaking away for a couple of hundred years, and very inexperienced to boot.  And getting him to the point where he's capable of doing a replication backup without incident.

In particular psy0rz and Scrin are due some heavy thanking; PsyOrz, for quickly going to the heart of things with very few words, and Scrin for thoughtful, helpful and well written coaching. so taking the moment to thank the forum and these two fellas in particular for their time and willingness to help.

I've made my first replication backup without incident or problems (well, except of my own making).  So now I'm ready to start tearing up the HP Z800, hopefully I will get it fixed and not need the backup I have waiting.

**Comment by psy0rz** [comment]
good luck :)
